datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -229,6 +249,11 @@ private static void configureDatasetResolvers(final RuntimeWiring.Builder builde
                                     DOWNSTREAM_LINEAGE_TYPE,
                                     (env) -> ((Dataset) env.getSource()).getUrn()))
                     )
+                    .dataFetcher("genericDownstreamLineage", new AuthenticatedResolver<>(
```

Comment:
Note to self: We can clean up "authenticated resolvers" now. All requests to GQL endpoints are protected by authentication today.